### PR TITLE
Studio dvcyaml

### PR DESCRIPTION
**Pull request by dberenbaum**
Make posts to studio drop `dvc.yaml::` from plots

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -137,4 +137,6 @@ dmypy.json
 # Cython debug symbols
 cython_debug/
 
+.dvc/
+.dvcignore
```
Comment (dberenbaum) [inline .gitignore:141]: Unrelated, but I keep getting these generated when running tests

**Review by daavoo — APPROVED** [review]
We should wait and bump min DVC, right?

**Comment by dberenbaum** [comment]
Yes, this depends on both https://github.com/iterative/dvc/pull/9898 and https://github.com/iterative/studio/pull/7628. We can either wait to merge it into the 3.0 branch, or merge now and wait for those changes to release 3.0.

**Comment by dberenbaum** [comment]
Actually, we don't technically need a dvc version bump in dvclive, at least for this PR, since this only affects Studio. Probably good to do it anyway to avoid weird plots issues in 3.0, so I added it here.

**Pull request merged**